project-base/storefront/components/Blocks/PromoCode/PromoCodeInfo.tsx

```diff
@@ -22,9 +22,11 @@ export const PromoCodeInfo: FC<PromoCodeInfoProps> = ({ onRemovePromoCodeCallbac
                 </LabelLink>
             </div>
             <p className="text-textDisabled ">
-                {t(
-                    'The discount was applied to all non-discounted items to which the promotion applies according to the rules.',
-                )}
+                {promoCode.type === TypePromoCodeTypeEnum.FreeTransportPayment
+                    ? t('The discount was applied to the order transport and payment.')
```

Comment:
I feel like we do not call it discount, as we have no discount order item, but we call it free transport and payment, so mb reword it:

```suggestion
                    ? t('Free transpot and payment was applied.')
```

Reply:
Honestly, I do not want to change it now, there is quite a lot of information regarding the free transport after the change. It is a translated message so it can be changed easily. The original "The discount was applied to the order transport and payment." message is closer to the other promo codes message, maybe, one day, we will have a design that answers the question of what should be here by default :smile: I would let @pk16011990 do the decision here now

Reply:
I wouldn't bother with the message anymore.